[HR Finances] A lot of changes

mkujda opened the pull request:
Its not a bug, its a feature.

Graphics, Forms.

annakowalczyk1 committed:
A

annakowalczyk1 committed:
2

annakowalczyk1 committed:
3

annakowalczyk1 committed:
1

annakowalczyk1 committed:
1

annakowalczyk1 committed:
1

MichalBrr committed:
Praca sobotnia

annakowalczyk1 committed:
1

KubaBartkiewicz committed:
Duzo_zmian

KubaBartkiewicz committed:
Merge branch 'HRFinanse1' of https://github.com/rudnickirobert/KWZP2019 into HRFinanse2

# Conflicts:
#	KWZP2019/KWZP2019/FinancesAndHR.Designer.cs
#	KWZP2019/KWZP2019/KWZP2019.csproj

KubaBartkiewicz committed:
nowe_pliki

mkujda committed:
Merge branch 'Finanse-ciag-dalszy-pracy' into HRFinanse2

Resolved conflicts

annakowalczyk1 committed:
1

mkujda committed:
Merge remote-tracking branch 'origin/HRFinanse2' into HRFinanse2

MichalBrr committed:
Filtrowanie po datach w Profits nie działa

MichalBrr committed:
Obliczanie czystego zysku firmy

MichalBrr committed:
Formularz Profits sprawny, brak generowania raportu

MichalBrr committed:
Generowanie raportu, pełne sortowanie datami w każdym z pięciu datagridview.

MichalBrr committed:
Zmiany nazw formularzy, uzupełnianie funkcjonalności

annakowalczyk1 committed:
1

annakowalczyk1 committed:
Merge branch 'HRFinanse2' of https://github.com/rudnickirobert/KWZP2019 into HRFinanse2

mkujda committed:
Merge branch 'Finanse-ciag-dalszy-pracy' into HRFinanse2

mkujda committed:
[HRFinances]Fixed_compilation

KubaBartkiewicz committed:
kkkk

KubaBartkiewicz committed:
Merge branch 'HRFinanse2' of https://github.com/rudnickirobert/KWZP2019 into HRFinanse2

# Conflicts:
#	KWZP2019/KWZP2019/HR/Absences.Designer.cs

KubaBartkiewicz committed:
trurrr

mkujda committed:
Merge branch 'master' into HRFinanse2

KubaBartkiewicz committed:
poprawki

rudnickirobert commented on KWZP2019/KWZP2019/Finances/PaymentHistory.cs:
```diff
@@ -0,0 +1,77 @@
+    public partial class Salaries : Form
+    {
+        RoofingCompanyEntities db;
+        SqlConnection sqlConnection = new SqlConnection("server =.\\SQLEXPRESS;Database=RoofingCompany;Integrated Security=true");
```
mamy to w zmiennej `db`

rudnickirobert commented on KWZP2019/KWZP2019/Finances/PaymentHistory.cs:
```diff
@@ -0,0 +1,77 @@
+        private void Salaries_Load(object sender, EventArgs e)
+        {
+            SqlCommand sqlCommand = new SqlCommand
+            ("SELECT dbo.Employee.EmployeeName, dbo.Employee.EmployeeSurname, dbo.Employee.PESEL, " +
```
to może do widoku?

Kusik03 committed:
Dodano nowe inserty

mkujda committed:
HR_views

mkujda committed:
HR views and Entity Update

Kusik03 committed:
Dodane View Finanse

mkujda committed:
HR_Finances_Fixed_Compilation

mkujda committed:
Merge remote-tracking branch 'origin/New_inserts_HR' into HRFinanse2

Merge all views and resolved conflicts

mkujda committed:
HR_Fixed views

mkujda committed:
HR_Finances_AllViews_EntityUpdate

MichalBrr committed:
[Finanse] Modyfikacja wyglądu okien finanse

MichalBrr committed:
[Finanse] Okno AllIncomes sprawne, bez funkcjonalności wyszukiwania

MichalBrr committed:
[HRiFinanse_Finanse] Added functionalities of following tables:

ProfitsAndFinancialBalance
PaymentHistory
Finances
AllIncomes
AllExpences
AddNewCompany

mkujda committed:
Finances_vInvoiceTypes_EntityUpdate

mkujda committed:
HR_Views_Entity

Views for Medical Examination and Contract End Alerts.
Updated Entity.

mkujda committed:
Replace duplicated contractor Fix entities data types

* Replace duplicated T-Mobile Contractor
* Update Entities Data-Types, after db changes type didn't update automatically from tinyint to int which caused arithmetic overflow (Table Positions)

MichalBrr committed:
[HRandFinances_Finances] All forms except SalariesAbsencesSummary are fully operational. Minor improvements and modyfications, label names and camelcase.

MichalBrr committed:
Merge remote-tracking branch 'origin/HRFinanse2' into HRFinanse2

mkujda committed:
Merge remote-tracking branch 'origin/HRFinanse2' into HRFinanse2

annakowalczyk1 committed:
Edycja tabeli w formularzu Nieobecnocci

annakowalczyk1 committed:
Edytowanie tabeli w formularzu Nieobecnosci cz2

MichalBrr committed:
[HRandFinances_Finances] Code cleared, improvements in switching from form to form added.

KubaBartkiewicz committed:
Changes in the way of displaying tables, to sql views

KubaBartkiewicz committed:
Merge branch 'HRFinanse2' of https://github.com/rudnickirobert/KWZP2019 into HRFinanse2

# Conflicts:
#	KWZP2019/KWZP2019/HR/HR.Designer.cs

annakowalczyk1 committed:
Przyciski powrót/powrót do menu

KubaBartkiewicz committed:
Changes in tbSearchingEmployees, added default value

annakowalczyk1 committed:
Edycja tabel

annakowalczyk1 committed:
Edycja formularza AddInvoices

KubaBartkiewicz committed:
Merge remote-tracking branch 'remotes/origin/HRFinanse3' into HRFinanse2

# Conflicts:
#	KWZP2019/KWZP2019/Finances/Finances.cs
#	KWZP2019/KWZP2019/HR/Absences.cs
#	KWZP2019/KWZP2019/HR/HR.Designer.cs

annakowalczyk1 committed:
Praca nad GUI

MichalBrr committed:
[HRandFinances] CamelCase fixes

MichalBrr committed:
Merge remote-tracking branch 'origin/HRFinanse2' into HRFinanse2

annakowalczyk1 committed:
Edycja widoku formularzy: AddInvoices, AddNewCompany, Absences

annakowalczyk1 committed:
Merge branch 'HRFinanse2' of https://github.com/rudnickirobert/KWZP2019 into HRFinanse2

KubaBartkiewicz committed:
Code formating

KubaBartkiewicz committed:
Merge branch 'HRFinanse2' of https://github.com/rudnickirobert/KWZP2019 into HRFinanse2

# Conflicts:
#	KWZP2019/KWZP2019/HR/Absences.cs

KubaBartkiewicz committed:
Errors Repaired

rudnickirobert commented:
@mkujda co z tym PR?

mkujda commented:
@rudnickirobert za Pana pozwoleniem zamknę, wszystkie zmiany zawarte tutaj znajdowały się w branchu który wszedł na mastera.

rudnickirobert commented:
@mkujda gdyby tak było to nie byłoby ani konfliktów ani zmian pokazanych przez githuba :)
ale na zamknięcie mogę się zgodzić- wtedy zauważy Pan czego brakuje
